packages/ui/src/components/Notice/Notice.tsx

```diff
@@ -0,0 +1,359 @@
+          >
+            {onDismiss && (
+              <IconButton
+                icon={XMarkIcon}
+                size="sm"
+                className="absolute top-2 right-2"
+                color={iconColor ?? "black"}
+                onClick={handleDismiss}
+                label={intl.formatMessage(uiMessages.closeAlert)}
+              />
+            )}
+            {children}
+          </div>
+        )}
+      </NoticeContext.Provider>
+    );
+  },
+);
+
+const title = tv({
+  slots: {
+    icon: "h-auto w-full",
+    heading: "mb-.25 col-start-2 font-bold",
+  },
+  variants: {
+    small: {
+      true: {
+        heading: "text-sm/6",
+      },
+      false: {
+        heading: "sm:leading-7",
+      },
+    },
+    color: {
+      gray: "",
+      primary: "",
+      secondary: "",
+      success: "",
+      warning: "",
+      error: "",
+    },
+  },
+  compoundSlots: [
+    {
+      slots: ["icon", "heading"],
+      color: "gray",
+      class: "text-gray-600 dark:text-gray-200",
+    },
+    {
+      slots: ["icon", "heading"],
+      color: "primary",
+      class: "text-primary-600 dark:text-primary-200",
+    },
+    {
+      slots: ["icon", "heading"],
+      color: "secondary",
+      class: "text-secondary-600 dark:text-secondary-200",
+    },
+    {
+      slots: ["icon", "heading"],
+      color: "success",
+      class: "text-success-600 dark:text-success-200",
+    },
+    {
+      slots: ["icon", "heading"],
+      color: "warning",
+      class: "text-warning-600 dark:text-warning-200",
+    },
+    {
+      slots: ["icon", "heading"],
+      color: "error",
+      class: "text-error-600 dark:text-error-200",
+    },
+  ],
+});
+
+const iconMap = new Map<RootVariants["color"], IconType>([
+  ["gray", BellAlertIcon],
+  ["primary", BellAlertIcon],
+  ["secondary", BellAlertIcon],
+  ["success", CheckCircleIcon],
+  ["warning", ExclamationCircleIcon],
+  ["error", ExclamationTriangleIcon],
+]);
+
+interface TitleProps {
+  icon?: IconType;
+  as?: HeadingRank | "p";
+  children: ReactNode;
+  defaultIcon?: boolean;
+}
+
+const Title = ({
+  icon: iconEl,
+  as: Heading = "p",
+  defaultIcon = false,
+  children,
+}: TitleProps) => {
+  const { small, color } = use(NoticeContext);
+  const { icon, heading } = title({ small, color });
+  let Icon = iconEl;
+  if (!iconEl && defaultIcon) {
+    Icon = iconMap.get(color);
+  }
+
+  return (
+    <>
+      {Icon && <Icon className={icon()} />}
+      <Heading className={heading()}>{children}</Heading>
+    </>
+  );
+};
+
+const Content = ({ children, className, ...rest }: DivProps) => (
+  <div {...rest} className={twMerge("col-start-2", className)}>
+    {children}
+  </div>
+);
+
+const actions = tv({
+  base: "col-start-2 flex flex-wrap items-center gap-6",
+  variants: {
+    small: {
+      true: "mt-3",
+      false: "mt-4.5",
+    },
+  },
+});
+
+const Actions = ({ className, ...rest }: DivProps) => {
```

Comment:
In mobile view, the actions should stack up I think.

<img width="1397" height="768" alt="Image" src="https://github.com/user-attachments/assets/7e29368a-cb79-446e-8354-fcce67e5ed5e" />